demo_fastapi/model.py

```diff
@@ -4,36 +4,39 @@
 
 
 class Order(BaseModel):
-    id: int
-    datestamp: str
-    buyer: str
+    id: int | None = None
+    datestamp: str | None = None
+    buyer: str | None = None
```

Comment:
Aren't timestamp and buyer required?

Reply:
So, in principle yes. I did it this way (with the check in app.post below) because I chose not to make another pydantic model for other functions, i.e. in app.put and some of my tests. When I use my model there, these two features are not always necessary to provide and it gives me ValidationError whenever I use an instance of the model without specifying value of these features. I couldn't find a way to circumvent validation only for certain usage of the model, so I decided to just test for NoneType in app.post, because that's the only place I had a requirement for not-None values. Feel free to give me a different suggestion how to do this!

Reply:
hmm I see !